patternfly-sass.gemspec

```
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/Patternfly/patternfly'
   s.license  = 'Apache-2.0'
 
-  s.add_runtime_dependency 'sass', '~> 3.4.15'
+  s.add_runtime_dependency 'sassc', '~> 3.6.1'
```

sassc is `2.2.1` ATM newest version, so this is IMHO wrong.
```suggestion
  s.add_runtime_dependency 'sassc', '~> 2.0'
```